runtime/tests/jvmtitests/src/com/ibm/jvmti/tests/decompResolveFrame/decomp005.c

```diff
@@ -0,0 +1,102 @@
+/*******************************************************************************
+ * Copyright (c) 2020, 2020 IBM Corp. and others
+ *
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/
+ * or the Apache License, Version 2.0 which accompanies this distribution and
+ * is available at https://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * This Source Code may also be made available under the following
+ * Secondary Licenses when the conditions for such availability set
+ * forth in the Eclipse Public License, v. 2.0 are satisfied: GNU
+ * General Public License, version 2 with the GNU Classpath
+ * Exception [1] and GNU General Public License, version 2 with the
+ * OpenJDK Assembly Exception [2].
+ *
+ * [1] https://www.gnu.org/software/classpath/license.html
+ * [2] http://openjdk.java.net/legal/assembly-exception.html
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
+ *******************************************************************************/
+
+#include "jvmti_test.h"
+#include <string.h>
+
+static agentEnv * env;
+static int imse = 0;
+ 
+static void JNICALL
+cbSingleStep(jvmtiEnv *jvmti_env, JNIEnv* jni_env, jthread thread, jmethodID method, jlocation location)
+{
+
+}
+
+
+jint JNICALL
+decomp005(agentEnv * _env, char * args)
+{
+	JVMTI_ACCESS_FROM_AGENT(_env);
+	jvmtiError err;                	
+	jvmtiEventCallbacks callbacks;
+	jvmtiCapabilities capabilities;                
+
+	env = _env;
+
+	if (!ensureVersion(env, JVMTI_VERSION_1_1)) {
+		return JNI_ERR;
+	}   
+    
+	if (NULL != args) {
+		if (0 == strcasecmp(args, "imse")) {
+			imse = 1;
+		} else {
+			error(env, err, "Unknown option");
```

Comment:
@gacholio this is failing to compile on Windows.
`warning C4700: uninitialized local variable 'err' used`

Reply:
Created https://github.com/eclipse/openj9/pull/9544 to address the compile issue